### ISSUE TITLE
what's the meaning are in the command?

### ISSUE THREAD

**Issue by noticeable** [y2l/social-relation-tensorflow#1]
hello
       thx for you share of your code.I'm going to cite your parper for a further reacher.however, when I  use the comman:
python main.py --train = True --net_arch = vgg19 --double_stream_mode = True --epoch_num = 10 --batch_size = 10 --cls_num = 16 --learning_rate = 1e-4 --shuffle_dataset = True --img_list1 = $ {DATA_LIST1} --img_list2 = $ {DATA_LIST2}
to train the net. there is show a error about DATA_LIST1 and DATA_LIST2.I have download all resource you said in Readme.and I have tried to use like : 
python main.py --train=True --net_arch=vgg19 --double_stream_mode=True --epoch_num=10 --batch_size=10 --cls_num=16 --learning_rate=1e-4 --shuffle_dataset=True --img_list1=${./all_single_face} --img_list2=${./all_single_body}

but it show errors too.could you please tell what's mean is it?  

Looking forward to your reply.

**Comment by yaoyao-liu** [comment]
Hi Li,

Thanks for your attention on this project.

DATA_LIST1 and DATA_LIST2 are not the real filenames. If you want to run the code, you should replace them with the directories of your downloaded data list files.
For example, If you download the splits in your current folder, the command must be changed to:

python main.py --train=True --net_arch=vgg19 --double_stream_mode=True --epoch_num=10 --batch_size=10 --cls_num=16 --learning_rate=1e-4 --shuffle_dataset=True --img_list1=./train_test_eval_splits/relation_consistency3/single_body1_train_16.txt --img_list2=./train_test_eval_splits/relation_consistency3/single_body1_train_16.txt


​
As for how to choose the datalist, you may read the paper provided on the GitHub and this project page:
https://www.mpi-inf.mpg.de/departments/computer-vision-and-multimodal-computing/research/human-activity-recognition/social-relation-recognition/

If you have any further questions, feel free to contact me. You may directly send me email: yaoyaoliu@outlook.com

**Comment by noticeable** [comment]
thx for your reply.According to your method, I have solved this problem.however ,I have met another question about your Item,it is not  related to this quetion.so I will close this question.I will send a email about it to you.wish your reply

**Comment by yaoyao-liu** [comment]
> thx for your reply.According to your method, I have solved this problem.however ,I have met another question about your Item,it is not related to this quetion.so I will close this question.I will send a email about it to you.wish your reply

Okay. I'll close this issue. It's both okay for me to contact with you via github issue and email.

If you get further questions, feel free to create another issue.
